src/software/thunderscope/replay/proto_player.py

```diff
@@ -405,14 +408,19 @@ def __play_protobufs(self) -> None:
 
                 with self.replay_controls_mutex:
 
-                    # Unpack the current entry in the chunk
-                    (
-                        self.current_packet_time,
-                        proto_class,
-                        proto,
-                    ) = ProtoPlayer.unpack_log_entry(
-                        self.current_chunk[self.current_entry_index]
-                    )
+                    try:
+                        # Unpack the current entry in the chunk
+                        (
+                            self.current_packet_time,
+                            proto_class,
+                            proto,
+                        ) = ProtoPlayer.unpack_log_entry(
+                            self.current_chunk[self.current_entry_index]
+                        )
+                    except ValueError:
+                        self.current_entry_index += 1
+                        logging.error("[ProtoPlayer] Error parsing log entry")
+                        continue
```

Comment:
Running the AI vs AI replay from CI, I got ~20 of this log spamming. Do you know why this is happening now?

Reply:
I also saw this error and it happens when loading the last few chunks in a replay log. I I think it's the same issue as I mentioned above. We're killing the process aggressively with SIGTERM so the last chunk may not have written to disk correctly.

Reply:
Yeah it seems to only happen at the end of the replay. Also, the CI replay is only 12 seconds, likely due to the compilation taking most of the time. You may want to increase the timeout, or use an alternative way to measure the time.

Reply:
good catch I don't think the compilation is the issue here because I built thunderscope_main in the previous step. Either:
- the 2-core container is super slow and that 3-minute async mode is actually slower than real-time (because it is waiting to receive primitive sets from both AIs before advancing time; on our local fast machine we assume that ssl vision is the bottleneck, not AI so async mode should tick way faster)
- or the SIGKILL is preventing the protologger from dumping the logs to the file cleanly

I re-wrote the thunderscope killing logic into thunderscope_main to make the end of ci_mode a bit more elegant and hopefully prevent these two issues